test/e2e/testing-manifests/ingress/static-ip-2/ing.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
```

Comment:
`nginx` seems to have its own folder under `test/e2e/testing-manifests`. Should GCE do the same since we'll be testing annotations specific to this controller?

Reply:
Will make that change in a separate PR.

Reply:
See issue #58640